engine/src/platform/platform_linux.c

```diff
@@ -10,10 +10,8 @@
 #include "containers/darray.h"
 
 #include <xcb/xcb.h>
-#include <X11/keysym.h>
-#include <X11/XKBlib.h>  // sudo apt-get install libx11-dev
-#include <X11/Xlib.h>
-#include <X11/Xlib-xcb.h>  // sudo apt-get install libxkbcommon-x11-dev libx11-xcb-dev
+#include <xcb/xcb_keysyms.h>
```

Comment:
Sorry I'm new to xcb, does it require any additional packages installed or does it come as a standard package? If it does require any package installed, it would be greate to leave the install command like it was there before :)